.github/workflows/generate_manpage.yml

```
@@ -19,7 +19,35 @@ jobs:
           python-version: 3.x
 
       - name: Install dependencies
-        run: pip install argparse-manpage
+        run: |
+          pip install argparse-manpage
+          pip install six
+          pip install pexpect
+          pip install rpm
```

Try combining all of them into one single command, like: 

```suggestion
          pip install argparse-manpage six pexpect rpm
```

---

Thank you for this help with the format on this install command. I didn't know you could set it up like this